usacloud load-balancer vip-add でSIGSEGV

vip-addを行ったところ下記の様にSIGSEGVが発生します。
0.6.0 (とpre-release版両方) / mac OS Sierra で試しました。
Name or ID 意外に --selector で指定しても同じ結果でした。

パラメータの指定が間違っているのでしょうか？
よろしくお願いいたします。

```
usacloud load-balancer vip-add --vip (IP) --port 80 -y "LB NAME"
```

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x17b177b]

goroutine 65 [running]:
github.com/sacloud/usacloud/command/funcs.LoadBalancerVipAdd(0x2b59620, 0xc42008b630, 0xc4206b8200, 0x0, 0x0)
	/go/src/github.com/sacloud/usacloud/command/funcs/load_balancer_vip_add.go:20 +0x9b
github.com/sacloud/usacloud/command/cli.init.17.func26.1(0x2b59620, 0xc42008b630, 0xc4206b8200, 0xc42058c620, 0xc4202daa40)
	/go/src/github.com/sacloud/usacloud/command/cli/cli_load_balancer_gen.go:4379 +0x46
created by github.com/sacloud/usacloud/command/cli.init.17.func26
	/go/src/github.com/sacloud/usacloud/command/cli/cli_load_balancer_gen.go:4378 +0xfb2
```

--- a/command/cli/cli_load_balancer_gen.go
+++ b/command/cli/cli_load_balancer_gen.go
@@ -1510,7 +1510,7 @@ func init() {
 					&cli.BoolFlag{
 						Name:    "force",
 						Aliases: []string{"f"},
-						Usage:   "forced-shutdown flag if server is running",
+						Usage:   "forced-shutdown flag if load-balancer is running",
 					},
 					&cli.StringSliceFlag{
 						Name:  "selector",

--- a/command/funcs/load_balancer_functions.go
+++ b/command/funcs/load_balancer_functions.go
@@ -1,0 +1,12 @@
+package funcs
+
+import "github.com/sacloud/libsacloud/sacloud"
+
+func initLoadBalancerSettings(lb *sacloud.LoadBalancer) {
+	if lb.Settings == nil {
+		lb.Settings = &sacloud.LoadBalancerSettings{}
+	}
+	if lb.Settings.LoadBalancer == nil {
+		lb.Settings.LoadBalancer = []*sacloud.LoadBalancerSetting{}
+	}
+}

--- a/command/funcs/load_balancer_vip_add.go
+++ b/command/funcs/load_balancer_vip_add.go
@@ -16,6 +16,8 @@ func LoadBalancerVipAdd(ctx command.Context, params *params.VipAddLoadBalancerPa
 		return fmt.Errorf("LoadBalancerVipAdd is failed: %s", e)
 	}
 
+	initLoadBalancerSettings(p)
+
 	// validate
 	if len(p.Settings.LoadBalancer) >= 4 {
 		return fmt.Errorf("LoadBalancer already has maximum count of VIP")

--- a/command/funcs/load_balancer_vip_delete.go
+++ b/command/funcs/load_balancer_vip_delete.go
@@ -15,6 +15,8 @@ func LoadBalancerVipDelete(ctx command.Context, params *params.VipDeleteLoadBala
 		return fmt.Errorf("LoadBalancerVipDelete is failed: %s", e)
 	}
 
+	initLoadBalancerSettings(p)
+
 	// index
 	if params.Index <= 0 || params.Index-1 >= len(p.Settings.LoadBalancer) {
 		return fmt.Errorf("index(%d) is out of range", params.Index)

--- a/command/funcs/load_balancer_vip_info.go
+++ b/command/funcs/load_balancer_vip_info.go
@@ -14,6 +14,7 @@ func LoadBalancerVipInfo(ctx command.Context, params *params.VipInfoLoadBalancer
 	if e != nil {
 		return fmt.Errorf("LoadBalancerVipInfo is failed: %s", e)
 	}
+	initLoadBalancerSettings(p)
 
 	vips := p.Settings.LoadBalancer
 	if len(vips) == 0 {

--- a/command/funcs/load_balancer_vip_update.go
+++ b/command/funcs/load_balancer_vip_update.go
@@ -14,6 +14,7 @@ func LoadBalancerVipUpdate(ctx command.Context, params *params.VipUpdateLoadBala
 	if e != nil {
 		return fmt.Errorf("LoadBalancerVipUpdate is failed: %s", e)
 	}
+	initLoadBalancerSettings(p)
 
 	// validate
 

--- a/define/load_balancer.go
+++ b/define/load_balancer.go
@@ -515,7 +515,7 @@ func loadBalancerDeleteParam() map[string]*schema.Schema {
 			Type:        schema.TypeBool,
 			HandlerType: schema.HandlerNoop,
 			Aliases:     []string{"f"},
-			Description: "forced-shutdown flag if server is running",
+			Description: "forced-shutdown flag if load-balancer is running",
 			Category:    "load-balancer",
 			Order:       10,
 		},

--- a/test/integration/bats/load-balancer/core-commands.bats
+++ b/test/integration/bats/load-balancer/core-commands.bats
@@ -1,0 +1,56 @@
+#!/usr/bin/env bats
+
+load ${BASE_TEST_DIR}/helpers.bash
+
+resource_name=${TEST_TARGET_NAME}01
+
+@test "Usacloud: should can create load-balancer" {
+  # create Switch
+  run usacloud_run switch create --name "$resource_name" -y -q
+
+  [ -n "${output}" ]
+  [ ${status} -eq 0 ]
+
+  switch_id=$(echo ${output})
+
+  run usacloud_run load-balancer create -y -q \
+          --name "$resource_name" \
+          --switch-id "$switch_id" \
+          --ipaddress1 "192.168.100.101" \
+          --nw-mask-len 24
+
+  [ -n "${output}" ]
+  [ ${status} -eq 0 ]
+
+}
+
+@test "Usacloud: should can add VIP" {
+
+  lb_id=$(usacloud_run load-balancer read -q "$resource_name")
+
+  # run vip-add
+  run usacloud_run load-balancer vip-add --vip 192.168.100.1 --port 80 -y "$resource_name"
+  [ "${status}" -eq 0 ]
+
+  # run vip-update
+  run usacloud_run load-balancer vip-update --index 1 --vip 192.168.100.2 -y "$resource_name"
+  [ "${status}" -eq 0 ]
+
+  # run vip-del
+  run usacloud_run load-balancer vip-delete --index 1 -y "$resource_name"
+  [ "${status}" -eq 0 ]
+}
+
+@test "Usacloud: should can delete load-balancer" {
+
+  run usacloud_run load-balancer rm -f -y "${resource_name}"
+
+  [ -n "${output}" ]
+  [ ${status} -eq 0 ]
+
+  run usacloud_run switch rm -y "${resource_name}"
+
+  [ -n "${output}" ]
+  [ ${status} -eq 0 ]
+
+}

